engine.py

```diff
@@ -112,18 +112,22 @@ def get_knight_moves(self, r, c, moves):
 
 	def get_king_moves(self, r, c, moves):
  		##TODO
+		rows = (r-1, r, r+1)
+		columns = (c+1, c, c-1)
```

Comment:
Make the two tuples uniform, I.e

rows = (r-1, r, r+1)
columns = (c-1, c, c+1)

It won't affect performance, but I think it'll make your code more readable